src/core.c/Real.pm6

```diff
@@ -148,7 +148,7 @@ multi sub infix:<**>(Real \a, Real \b)  { a.Bridge ** b.Bridge }
 
 multi sub infix:«<=>»(Real \a, Real \b) { a.Bridge <=> b.Bridge }
 
-multi sub infix:<==>(Real \a, Real \b)  { a.Bridge == b.Bridge }
+multi sub infix:<==>(Real:D \a, Real:D \b)  { a.Bridge == b.Bridge }
```

Comment:
@lizmat BTW, can you have a look at this? I remember there was a discussion about using definites for ops, but don't remember the outcome of it. For now it's only `==` is changed to pass a spectest, but looks like the rest would have to be changed too.

Reply:
Looks to me they would all need a `:D`:

    $ r 'dd infix:<==>(Real,Real)'
    Invocant of method 'Bridge' must be an object instance of type 'Real',
    not a type object of type 'Real'.  Did you forget a '.new'?

    $ r 'dd infix:«<=>»(Real,Real)'
    Invocant of method 'Bridge' must be an object instance of type 'Real',
    not a type object of type 'Real'.  Did you forget a '.new'?